src/porepy/models/fluid_mass_balance.py

```diff
@@ -743,6 +722,26 @@ def set_discretization_parameters(self) -> None:
                 },
             )
 
+    def set_boundary_data(self) -> None:
```

Comment:
Should we call this from update_time_dependent_arrays?

Reply:
There's something weird with this method, as for my knowledge. A search through the project files shows that this method `update_time_dependent_ad_arrays` is called only in the poromechanics model. Should I add it to the common solution strategy?

Reply:
Do you mean the implementation? Let's discuss in person.

Reply:
Closing after discussion in person.